files-community/Files
# Code Quality: Removed Vanara from IStorageNetworkService

### Resolved / Related Issues

- Related #15000
- Related #14572

### Steps used to test these changes

- Go to Home
- Enable Network section
- See the enumerated network computers and shortcuts
